### ISSUE TITLE
Add-on doesn't support 2.1.31

### ISSUE THREAD

**Issue by ffuuugor** [matthayes/anki_progress_stats#29]
At some point around version 2.1.31 anki app redesigned stats page, and it no longer displays additional graphs from this plug-in

**Comment by Shelbinator94** [comment]
Yep, same problem! I miss these graphs, they're incredible! Honestly, they're the only stats I use, the vanilla stats aren't that helpful.

**Comment by matthayes** [comment]
Hi thanks for reporting this.  Supposedly if you shift+click on the stats button it shows the old graphs (see [here](https://changes.ankiweb.net/#/?id=changes-in-2128) for the changes in 2.1.28 where this is mentioned).  Can you try this and let me know if it works?  I haven't had a chance to try it yet.  Hopefully this will be a workaround until I migrate the stats to the new page.

**Comment by Shelbinator94** [comment]
Boy howdy that does work! It disables all my add-ons after I exit stats, but a quick restart fixes that. Thanks man!

On Sep 1, 2020 3:00 PM, Matthew Hayes <notifications@github.com> wrote:
This message was sent from a non-IU address. Please exercise caution when clicking links or opening attachments from external sources.


Hi thanks for reporting this. Supposedly if you shift+click on the stats button it shows the old graphs (see here<https://changes.ankiweb.net/#/?id=changes-in-2128> for the changes in 2.1.28 where this is mentioned). Can you try this and let me know if it works? I haven't had a chance to try it yet. Hopefully this will be a workaround until I migrate the stats to the new page.

-

**Comment by ghost** [comment]
<img width="1019" alt="Screenshot 2020-10-05 at 07 06 02" src="https://user-images.githubusercontent.com/1085562/95027093-4c238c00-06d9-11eb-85b9-b1028dabd39f.png">
I'm on Anki  2.1.34 and I can't get this add-on to appear. Any suggestions please? Thank you.

**Comment by matthayes** [comment]
@scokim did you try the shift+click suggestion above?

**Comment by ghost** [comment]
Yes that works with a new page opened up . I was wondering why it couldn't work on the home page ?

**Comment by matthayes** [comment]
The Anki stats page was redone and I haven’t migrated to the new one yet. The old one has been kept for now for backwards compatibility with plugins like this one.

**Comment by ghost** [comment]
Thank you very much.

**Comment by pleed8** [comment]
Hi I am still having an issue with the Progress stats bar add-on. I did try the suggested Shift+click, but the same bar graph statistics still appear instead of the one in the images by the add-on. I've also tried deleting/reinstalling add-on and downgrading to a different version of Anki with no luck either. 

<img width="1280" alt="Screen Shot 2021-07-28 at 10 08 53 AM" src="https://user-images.githubusercontent.com/88096950/127336955-75e7ad76-7852-4673-a4ef-9d13bb9fecb3.png">

**Comment by matthayes** [comment]
@pleed8 What version of Anki are you using and do you have any other plugins installed that modify the statistics page?  I'm using Anki 2.1.44 and it's working fine.  My stats page does not have a "Future Due" section of "Calendar".  If you are using other plugins that modify the stats page it's possible it is conflicting with mine by overriding the content.

**Comment by pleed8** [comment]
Hi Matthew,

I am currently using
Version ⁨2.1.44 (b2b3275f)⁩
Python 3.8.6 Qt 5.14.2 PyQt 5.14.2

See below of the current add-ons that I have:
- All the Settings below are that of the same exact settings in which my
friends have and they have the add-on just fine. I'm not exactly sure what
to do because I tried to read other posts before contacting you. Sorry for
the inconvenience.

**Comment by matthayes** [comment]
@pleed8 I think the issue is you are getting the new stats page instead of the old one where the plugin renders the graphs.  The new one has "Future Due" where the old one has "Forecast".  Holding down SHIFT while you click Stats should work.  What OS are you using?  Maybe it behaves differently on your machine.

**Comment by pleed8** [comment]
It worked . Thanks . Is there any way to permanently have the graph shown
or would I have to manually do it each time I go into the stats browser. I
am currently using MacOS Big Sur. Version 11.4

**Comment by matthayes** [comment]
I don't know if there's a way to have the old stats page permanently show up.  You'd have to hold shift each time you want to view these stats.

**Comment by pleed8** [comment]
Ok, thanks.
